### PR TITLE
build: remove unused common dependencies

### PR DESCRIPTION
**Pull request by maikelvdh**

### CODE REVIEW

File: CocoaPods/Podfile.lock [hunk 1]
```diff
@@ -208,53 +206,8 @@ DEPENDENCIES:
 
 SPEC REPOS:
   https://repositories.tomtom.com/artifactory/api/pods/cocoapods:
-    - TomTomSDKBindingFrameworkHTTPInternal
```
Comment (AlinaUshakova-TomTom) [inline CocoaPods/Podfile.lock:211]: I am not sure, why those dependencies are gone?

Comment (maikelvdh) [reply CocoaPods/Podfile.lock:211]: This is an update to the spec repository section in the lockfile which is changed as part of using new version of cocoapods. The pods and dependencies are still listed as required in the lines above.

File: TomTomSDKExamples/Examples/UseCase/OnlineNavigationContent.swift [hunk 2]
```diff
@@ -14,8 +14,6 @@ import Foundation
 import SwiftUI
 
 // TomTomSDK modules
-import TomTomSDKCommon
```
Comment (muhammedtanriverdi-tomtom) [inline TomTomSDKExamples/Examples/UseCase/OnlineNavigationContent.swift:17]: These libraries are mentioned in the "[Building a navigation app](https://developer.tomtom.com/ios/navigation/documentation/use-cases/building-a-navigation-app)" use case in the developer portal.

File: CocoaPods/Podfile [hunk 3]
```diff
@@ -12,8 +12,6 @@ plugin 'cocoapods-art', :sources => [
 ]
 
 target 'TomTomSDKExamples' do
-  pod 'TomTomSDKCommon', '0.32.0'
```
Comment (muhammedtanriverdi-tomtom) [inline CocoaPods/Podfile:15]: These libraries are mentioned in the "[Building a navigation app](https://developer.tomtom.com/ios/navigation/documentation/use-cases/building-a-navigation-app)" use case in the developer portal.

File: TomTomSDKExamples/Examples/UseCase/OnlineNavigationContent.swift (hunk 2 above)
Comment (maikelvdh) [reply TomTomSDKExamples/Examples/UseCase/OnlineNavigationContent.swift:17]: On the portal they will be removed as part of the subsequent release. This is already internally addressed.

File: CocoaPods/Podfile (hunk 3 above)
Comment (maikelvdh) [reply CocoaPods/Podfile:15]: https://github.com/tomtom-international/tomtom-navigation-ios-examples/pull/47#discussion_r1399314516

**Review by muhammedtanriverdi-tomtom — APPROVED** [review]

**Review by AlexandruLazarescu-TomTom — APPROVED** [review]

**Review by AlinaUshakova-TomTom — APPROVED** [review]

**Pull request merged**